Quaver.Shared/Screens/Downloading/DownloadingScreen.cs

```diff
@@ -853,6 +853,9 @@ private void SetRichPresence()
         {
             try
             {
+                SteamManager.SetRichPresence("State", "In the menus");
+                SteamManager.SetRichPresence("Details", "Downloading Maps");
+
                 DiscordHelper.Presence.Details = "Downloading Maps";
                 DiscordHelper.Presence.State = "In the menus";
                 DiscordHelper.Presence.LargeImageText = OnlineManager.GetRichPresenceLargeKeyText(ConfigManager.SelectedGameMode.Value);
```

Comment:
It'd be better if we can change both Discord and Steam in one method, so we can avoid having to do this twice every single time.

Reply:
fair enough, I'll do that.